regression/CMakeLists.txt

```diff
@@ -37,10 +37,29 @@ function(add_esbmc_regression_test folder modes test)
       LABELS "regression;${folder}")
 endfunction()
 
+function(add_smtlib_regression_test folder modes test)
+    set(test_name "smtlib/${folder}/${test}")
+    add_test(NAME ${test_name}
+             COMMAND ${Python_EXECUTABLE} ${ESBMC_REGRESSION_TOOL}
+                     --tool=${ESBMC_BIN}
+                     --regression=${CMAKE_CURRENT_SOURCE_DIR}/${folder}
+                     --modes ${modes}
+                     --file=${test}
+                     --smt_test
+                     --smt_solver=z3
```

Comment:
This assumes that the executable `z3` is installed on the `UNIX AND NOT APPLE` system. For my PC that's true, but can we expect that for everyone running `ctest`?